### PR TITLE
remove needless map method

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: actionpack/lib/abstract_controller/rendering.rb [hunk 1]
```diff
@@ -78,11 +78,7 @@ def view_assigns
     # <tt>render :file => "foo/bar"</tt>.
     # :api: plugin
     def _normalize_args(action=nil, options={})
-      if action.is_a? Hash
-        action
-      else
-        options
-      end
+      action.is_a?(Hash) ? action : oprions
```
Comment (kaspth) [inline actionpack/lib/abstract_controller/rendering.rb]: This shouldn't be changed (and options is misspelled) :smile:

**Comment by kaspth** [comment]
Thanks :smile:

I'm borderlining on considering this a cosmetic commit, but I stumbled on the `map` the other day too and thought of removing it myself. I can't decide ¯\_(ツ)_/¯

**Comment by antnvtkvsk** [comment]
why do you need it?

**Comment by kaspth** [comment]
It's ostensibly a cosmetic commit in that it *mostly* reformats the code while blowing away potentially valuable git history. It does improve the readability a tiny bit by shifting to the more idiomatic %i shortcut.

You other change, however, is purely cosmetic and won't be accepted, so that has to go :smile:

**Comment by antnvtkvsk** [comment]
I TRIED! :lollipop:

**Comment by kaspth** [comment]
I'm sorry, but I don't understand what you're saying? :smile:

**Comment by kaspth** [comment]
@antnvtkvsk I see you've deleted your Rails fork. Is everything okay? :heart:

Though we might not have accepted this change, that doesn't mean you're a bad person or anything, so don't take any decision personally. Remember, you can always try again :heart:

**Comment by antnvtkvsk** [comment]
hah, it's ok. I just decided it can be changed, why not

**Comment by kaspth** [comment]
That's great! If you want more help please check out our contributing guidelines: http://edgeguides.rubyonrails.org/contributing_to_ruby_on_rails.html - it's jam-packed with all the things we like to see from contributors :grin:

**Pull request closed without merging**